<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutotuneSliders</class>
 <widget class="QWidget" name="AutotuneSliders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>557</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QLabel" name="lblDamp">
       <property name="text">
        <string>1.05</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Damping</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sensitivity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblNoise">
       <property name="text">
        <string>1.0 %</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="rateNoise">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sensitivity affects how much the quadcopter responds to both signal and noise from the sensors.&lt;/p&gt;&lt;p&gt;The default value of 1.0% is good for most uses. Increase this for snappier flight, but you may also need to increase damping slightly to prevent oscillation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="rateDamp">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Higher values of damping help prevent oscillations and make the quadcopter fly smoother, at the expense of slightly slower response.&lt;/p&gt;&lt;p&gt;The default value of 1.1 is generally a good value; increase to 1.3 to cure oscillation problems at high throttle.&lt;/p&gt;&lt;p&gt;(You can also slightly decreate the value of motor input output curve fit on the motors pane to reduce oscillations at high throttle.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="minimum">
        <number>85</number>
       </property>
       <property name="maximum">
        <number>135</number>
       </property>
       <property name="value">
        <number>110</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hovering the mouse on the above parameters will result in tooltips with more information.  Continuing with the default values is recommended.&lt;/p&gt;&lt;p&gt;The calculated values resulting from autotune are displayed below.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblWarnings">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnResetSliders">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Reset Parameters</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="10" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Altitude PosK&lt;span style=&quot; vertical-align:sub;&quot;&gt;p&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="lblAltPosKp">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="wn">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="cbUseYaw">
       <property name="text">
        <string>Yaw</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;RateK&lt;span style=&quot; vertical-align:sub;&quot;&gt;p&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="lblOuterKi">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="rollRateKi">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="lblOuterKp">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="pitchRateKd">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="pitchRateKi">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;RateK&lt;span style=&quot; vertical-align:sub;&quot;&gt;i&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="rollRateKd">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="rollRateKp">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="cbUseOuterKi">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Attitude K&lt;span style=&quot; vertical-align:sub;&quot;&gt;i&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="yawRateKp">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="yawRateKd">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;RateK&lt;span style=&quot; vertical-align:sub;&quot;&gt;d&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="yawRateKi">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="pitchRateKp">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Attitude K&lt;span style=&quot; vertical-align:sub;&quot;&gt;p&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="derivativeCutoff">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Natural frequency</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="cbTuneAlt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Altitude VelK&lt;span style=&quot; vertical-align:sub;&quot;&gt;p&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Altitude VelK&lt;span style=&quot; vertical-align:sub;&quot;&gt;i&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Derivative cutoff</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="lblAltVelKp">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLabel" name="lblAltVelKi">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="configgadget.qrc"/>
 </resources>
 <connections/>
</ui>
